Lecture2/Task5/Task5L2.java

```diff
@@ -0,0 +1,154 @@
+import java.io.*;
+import java.util.*;
+
+public class Task5L2 {
+
+    private static final String REGEX_NAME_LASTNAME = "[a-zA-Z]+";
+    private static final String MESSAGE_AGAIN = "Again? Really? Try something else :)";
+
+    private static BufferedReader reader = new BufferedReader ( new InputStreamReader ( System.in ) );
+
+    public static
+    void main ( String[] args ) throws Exception {
+        System.out.println ( "Let's start the game!!!" );
+        Console console = System.console ( );
+        String name = null;
+        while (name == null)
```

Comment:
you can reduce number of parameters that are required to be sent to the methods by assigning it to the class.

Reply:
fixed